provision/acc_provision/acc_provision.py

```diff
@@ -2184,6 +2230,10 @@ def provision(args, apic_file, no_random):
         err(" Dualstack feature is not supported. Please upgrade to Kubernetes version 4.21 or later or OpenShift version 4.8 or later.")
```

Comment:
Although not part of this patch, there is a typo here, kubernetes version should be 1.21

Reply:
addressed